core/src/main/java/net/adoptopenjdk/icedteaweb/client/console/JavaConsole.java

```diff
@@ -519,9 +480,9 @@ private void printThreadInfo() {
 
     public static void main(String[] args) {
 
-        final JavaConsole cconsole = new JavaConsole();
+        final JavaConsole console = new JavaConsole();
```

Comment:
isnt this nbow in clash with global variable? it would be unnecessary warning

Reply:
no, there is no field in this class which is called `console`